src/components/leaflet/sy-marker.tsx

```diff
@@ -0,0 +1,26 @@
+import SyMarkerImage from './sy-marker.png';
+import Leaflet from 'leaflet';
+
+const isBrowser = typeof window !== 'undefined';
+
+/**
+ * Leaflet doesn't support SSR, so we run into issues
```

Comment:
this took me some time. When the build first failed I chose to "quick fix" this with some if-else logic and push it without testing it locally until it failed on gatsby cloud. I tried this approach 4 or 5 times and then decided to do it properly. Being lazy is never good and this solution works.

The problem here is that we access the Leaflet Icon directly. React Leaflet is wrapping leaflet properly (it seems) but access Leaflet directly creates issues that we have to solve by our own.